ext/vscode/README.md

```diff
@@ -5,6 +5,9 @@ This extension makes it easier to run, create Azure Resources, and deploy Azure
 ## What It Does
 For more information about Azure Developer CLI and this VS Code extension, please [see the documentation](https://aka.ms/azure-dev/vscode).
 
+## Extension Documentation
+- [Available Commands](docs/commands.md) - Documentation for commands provided by this extension, including utility commands for use in VS Code configurations.
```

Comment:
I do not know if a relative link to another doc will work on the marketplace page or in VSCode itself--it needs to be tested.